documentation/modules/auxiliary/scanner/http/apache_mod_cgi_bash_env.md

```diff
@@ -92,3 +46,46 @@ printf "Test!\n"
 ```
 chmod +x /usr/lib/cgi-bin/test.sh
 ```
+
+### To check if a host is vulnerable to the attack
+1. run (on the host)
+```
+env 'x=() { :;}; echo vulnerable' 'BASH_FUNC_x()=() { :;}; echo vulnerable' bash -c "echo test"
+```
+2. The shell will return the below text if the environment is vulnerable
+``` 
+vulnerable
+bash: BASH_FUNC_x(): line 0: syntax error near unexpected token `)'   
+bash: BASH_FUNC_x(): line 0: `BASH_FUNC_x() () { :;}; echo vulnerable'
+bash: error importing function definition for `BASH_FUNC_x'
+test
+```
+
+## Verification Steps
+1. Do: run `msfconsole`
+2. Do: use `auxiliary/scanner/http/apache_mod_cgi_bash_env'
+2. Do: set `RHOSTS [IP]`
+3. Do: set `TARGETURI [URI]`
+4. Do: `run`
+
+## Options
+1. `CMD`. The default setting is /usr/bin/id
+2. `CVE`. The default setting is `CVE-2014-6271` but valid options are CVE-2014-6271 or CVE-2014-6278
+3. `HEADER`. The default setting is User-Agent
+4. `METHOD`. The default setting is GET
```

Comment:
Please see the [documentation](https://github.com/rapid7/metasploit-framework/wiki/Writing-Module-Documentation) [template](https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/module_doc_template.md) for example formatting.